commandline/src/main/java/tech/pegasys/web3signer/commandline/PicoCliSlashingProtectionParameters.java

```diff
@@ -92,6 +92,22 @@ public class PicoCliSlashingProtectionParameters implements SlashingProtectionPa
       arity = "1")
   boolean pruningAtBootEnabled = true;
 
+  @Option(
+      names = {"--slashing-protection-db-health-check-timeout-milliseconds"},
+      paramLabel = "<timeout in milliseconds>",
+      description =
+          "Number of milliseconds after which the database health check will be failed (Default: ${DEFAULT-VALUE})",
+      arity = "1")
+  private long dbHealthCheckTimeoutMilliseconds = 3000;
+
+  @Option(
+      names = {"--slashing-protection-db-health-check-interval-milliseconds"},
+      paramLabel = "<interval in milliseconds>",
+      description =
+          "Number of milliseconds between the database health check operation (Default: ${DEFAULT-VALUE})",
+      arity = "1")
+  private long dbHealthCheckIntervalMilliseconds = 3000;
```

Comment:
Default for the interval is probably a bit short. I think that 30s would be a more typical value.